packages/storybook/cxl-lumo-styles.stories.js

```diff
@@ -32,6 +32,37 @@ export const Typograhy = () => {
   `;
 };
 
+export const TypograhyUseCase = () => {
+  const override = boolean('Ignore media query', false);
+
+  return html`
+    <style>
+      div[override] {
+        --cxl-lumo-font-size-hero: 4rem !important;
+      }
+
+      div[override] h2 {
+        font-size: var(--lumo-font-size-xxxl) !important;
+      }
+
+      h3 {
+        font-size: var(--lumo-font-size-xxl) !important;
+      }
+
+      h4 {
+        font-size: var(--lumo-font-size-xl) !important;
+      }
+    </style>
+    <div ?override=${override}>
+      <h1>Become great at <strong>content marketing research</strong></h1>
```

Comment:
There could perhaps be even more "problematic" superlong-word typography examples here.

Perhaps @martinbon could help collect from Institute content or cxl.com/blog/

Let's do sample size n=5?

Reply:
**Additional samples:**
Digital psychology & behavioral design training
How to Design, Roll Out, & Scale an Optimization Program
Heuristic Analysis frameworks for conversion optimization audits
Implementing Urgency on eCommerce Product Pages For a 27.1% Lift [Case Study]
Checkout Optimization: How Do Trust Seals Affect Security Perception? [Original Research]

Reply:
Added those to the story. Looks ok to me.